addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -46,7 +46,7 @@
                             <button class="o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments btn btn-link" type="button" t-att-disabled="chatterTopbar.chatter.isDisabled" t-on-click="chatterTopbar.chatter.onClickButtonAttachments" title="Show Attachments">
                                 <i class="fa fa-paperclip"/>
                                 <t t-if="!chatterTopbar.chatter.isShowingAttachmentsLoading">
-                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount pl-1" t-esc="chatterTopbar.chatter.thread ? chatterTopbar.chatter.thread.allAttachments.length : 0"/>
+                                    <span class="o_ChatterTopbar_buttonCount o_ChatterTopbar_buttonAttachmentsCount pl-1" t-esc="chatterTopbar.chatter.thread and !chatterTopbar.chatter.thread.isFileBrowserOpen ? chatterTopbar.chatter.thread.allAttachments.length + ' files' : 'Attach files'"/>
```

Comment:
confusing variable name (cf other comment)

wrong translation of value, need to make a compute field for the "x files" part with 0, 1, 2, multi translation possibilities.